.github/workflows/stale-pr.yml

```diff
@@ -7,11 +7,12 @@ jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@v3
+      - uses: actions/stale@v9
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-pr-message: 'This PR was marked stale due to lack of activity. It will be closed in 7 days.'
+          stale-pr-message: 'This PR was marked stale due to lack of activity. It will be closed in 7 days. Consider applying the label `work in progress` to avoid this being marked as stale again.'
           close-pr-message: 'Closed as inactive. Feel free to reopen if this PR is still being worked on.'
-          exempt-pr-labels: 'release:after-ga'
+          exempt-pr-labels: 'bug,work in progress,experts needed'
```

Comment:
Would this need quoting/escaping for the spaces?

Reply:
Doesn't seem so by looking at examples in other repos

Reply:
It says a "comma separated list" so I think that should work. https://github.com/actions/stale?tab=readme-ov-file#exempt-pr-labels